grafana/grafana
# Elasticsearch: Add log highlight processing on backend

**What is this feature?**

Related to https://github.com/grafana/grafana/pull/63647. When processing log result on backend, we were missing implementation for log highlight and adding searchWords to log dataFrame. This PR fixes it.

To test this:

1. Run make `devenv sourcers=elastic`
2. Open Explore and run log query
3. Add `"log"` `"text"` or anything else as lucene query
4. Verify that highlight was correctly applied
